javascript/javascript1/week3/notepad.js

```diff
@@ -0,0 +1,36 @@
+const notes = [];
+
+function saveNote(content, id) {
+    let obj ={content: content, id: id }
```

Comment:
An id is by deifinition something used to uniquely identify an item. Consider what happens when we add  another item with the same '1'. How will this affect your code.

Reply:
it will add another note with different content with same id...

Reply:
0: {content: "Pick up groceries", id: 1}
1: {content: "Do laundry", id: 2}
2: {content: "Do laundghjgjh", id: 2}

Reply:
then you will have two entries with the same id! in a real life scenario that should never occur. How can you fix this.

Reply:
i fixed it ... thank you